Formula/qt.rb

```diff
@@ -109,9 +161,12 @@ def install
       include.install_symlink f/"Headers" => f.stem
     end
 
-    bin.glob("*.app") do |app|
-      libexec.install app
-      bin.write_exec_script libexec/app.basename/"Contents/MacOS"/app.stem
+    if OS.mac?
+      mkdir libexec
+      Pathname.glob("#{bin}/*.app") do |app|
+        mv app, libexec
+        bin.write_exec_script "#{libexec/app.stem}.app/Contents/MacOS/#{app.stem}"
+      end
```

Comment:
Why change the code after wrapping it in `if OS.mac?`

Reply:
Whoops, I think this got bungled because of a merge conflict from another PR that I didn't fully handle right.  I will push the fix for this along with any other requested changes.